### PR TITLE
MBS-11263 fix k8s get pods request metadata; add verbose logging

### PR DESCRIPTION
**Pull request by dsvoronin**
Almost ready sample project; still some problems with getting successful suite run

### CODE REVIEW

File: samples/gradle.properties [hunk 1]
```diff
@@ -1,3 +1,6 @@
+avito.logging.verbosity=DEBUG
+avito.build=local
```
Comment (dsvoronin) [inline samples/gradle.properties:2]: @sboishtyan That was the reason for "teamcity" deployment label. new project and i forgot to set this flag. We even have TODO message about local by default

File: samples/test-runner/build.gradle.kts [hunk 2]
```diff
@@ -29,12 +32,16 @@ android {
...
     androidTestImplementation("com.avito.android:test-inhouse-runner")
+    androidTestImplementation("com.avito.android:test-report")
+    androidTestImplementation("com.avito.android:logger")
     androidTestImplementation(libs.truth)
 
     androidTestUtil(libs.testOrchestrator)
 }
 
 instrumentation {
+    output = "$buildDir/reports/test-runner"
```
Comment (dsvoronin) [inline samples/test-runner/build.gradle.kts:43]: using `buildDir` is much much better choice for local development

File: samples/test-runner/build.gradle.kts [hunk 3]
```diff
@@ -73,8 +80,8 @@ instrumentation {
 
                     testsCountBasedReservation {
                         device = emulator29
-                        maximum = 2
-                        minimum = 1
```
Comment (dsvoronin) [inline samples/test-runner/build.gradle.kts:77]: We have some weird bug with 1 min device. Deployments generation never succeed with it

File: samples/test-runner/build.gradle.kts (hunk 2 above)
Comment (eugene-krivobokov) [reply samples/test-runner/build.gradle.kts:43]: We can use it for CI build also

File: samples/test-runner/build.gradle.kts (hunk 3 above)
Comment (eugene-krivobokov) [reply samples/test-runner/build.gradle.kts:77]: File an issue, please

File: samples/test-runner/build.gradle.kts (hunk 2 above)
Comment (dsvoronin) [reply samples/test-runner/build.gradle.kts:43]: yep

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**